.travis.yml

```diff
@@ -1,13 +1,11 @@
-sudo: required
```

Comment:
this has been deprecated, https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration